### ISSUE TITLE
Popup that opens on some sites

### ISSUE THREAD

**Issue by Dadan59DK** [ragibkl/adblock-dns-server#6]
Hello, 

I just discovered your DNS and I'm quite surprised at how fast it is. I am in France and I would like to know if it is possible to block popups on the following site: 

-https://www.ddl-island.co/index.php

Also, you can add these lists to your DNS server: 

-https://raw.githubusercontent.com/ookangzheng/blahdns/master/hosts/rpz.blacklist

- https://raw.githubusercontent.com/jerryn70/GoodbyeAds/master/Hosts/GoodbyeAds.txt

- https://raw.githubusercontent.com/EnergizedProtection/block/master/blu/formats/hosts

and can you add these domains to be blocked: 

- cxp.emogi.com
- us-central1-gcp.api.snapchat.com
- app-analytics.snapchat.com
- mvm.snapchat.com

Thank you

**Comment by ragibkl** [comment]
Hi @Dadan59DK ,

I'm glad that you found my dns servers useful. If you experience delays using them from france, you could try to set up your own on servers closer to you. You can use this repo as your base.

With regard to the popup, it seems like most of the ads redirect to sites in the domain `0byv9mgbn0.com`. It seems to be common in some dns list, but its not in mine. I can add them in later. However, there is no way to completely remove the popup actions on the click event. Those are done using javascript in the browser. You need to use some browser extensions that can remove those scripts for you.

The dns lists that you linked, I have to test them first, to make sure they are compatible with my scripts.

With regard to the domains you listed, why do you want those domains blocked? They look like legitimate API domains for snapchat. I'm afraid to block them, because I worry Snapchat may stop working for some users.

**Comment by Dadan59DK** [comment]
Thank you for your answer, 

it's a lot of snapchat api, snapchat ads come from these lists, then I tested via an application that takes blokada that is available on android. I let you test, if you ever need it, there is this site where there are very good filters: https://filterlists.com . 

If I can help you, I am a computer troubleshooter, I can always install your dns on my customers' PCs if you are interested of course.

**Comment by ragibkl** [comment]
Hi @Dadan59DK ,

> I let you test, if you ever need it, there is this site where there are very good filters: https://filterlists.com 

I did come across this site before. There are so many filters there, so I'm not sure which to pick. I cannot include everything, my servers capacity is quite small. Also, I don't want to block legitimate sites, so need to choose carefully. Whenever you have any ideas, you can help me choose from that site.

```
https://raw.githubusercontent.com/ookangzheng/blahdns/master/hosts/rpz.blacklist
https://raw.githubusercontent.com/jerryn70/GoodbyeAds/master/Hosts/GoodbyeAds.txt
https://raw.githubusercontent.com/EnergizedProtection/block/master/blu/formats/hosts
```
I've added these to my servers. My list just grew into > 1.5 million domains because of that. Ha ha, that was unexpected. All seems to work so far. Let me know if your experience is better after the change.

Interestingly, I tried using the site `https://www.ddl-island.co/index.php` after the dns changes. Seems like the popups stopped appearing. Please test on your end, to check if you get similar result.

Regarding the snapchat api domains you sent, let's review them first. If you can confirm to me within a few days, that it does not break user functions on the app, I will consider adding them.

```
If I can help you, I am a computer troubleshooter, I can always install your dns on my customers' PCs if you are interested of course.
```
I think this is entirely up to you. If you think the customer will benefit from using my dns, go ahead. There is a risk that they might experience problems when using my dns, whenever I make any changes. So, you need to be able to help them reverse the dns install. Otherwise, I welcome any users, feedback and test reports.

**Comment by Dadan59DK** [comment]
Thanks for your feedback, but the adult ads on https://www.ddl-island.co don't get blocked:/, I don't know if you can do anything about it, but the popups don't open anymore. Which is great:), also google "ads" are not blocked on google.fr:/ . I know that on the site I gave you, a filter blocks them. After that, if I can help you, there's no problem for me.

**Comment by ragibkl** [comment]
@Dadan59DK ,

I think I found the porn domain for that site. `cdn.emule-island.ru`.
Also, for google.fr, it is probably `adservice.google.fr`. I tried adding it, but it probably won't work.

I've added them to my custom list. You can help me test the sites.

```
I know that on the site I gave you, a filter blocks them
```
If you know which filter I should include, do let me know. I'm open to suggestion.

I think this issue can be marked as resolved. If you have any other issues, or suggestions, please create a new issue.

For now, thank you for the feedback and suggestions!!!
